### ISSUE TITLE
Servers

### ISSUE THREAD

**Issue by ghost** [CodePadApp/codepad-issues#72]
This ain't really a issue but a suggestion; allow codepad  to be able to run servers: java, node.js, python, PHP.

**Comment by Bongs237** [comment]
good idea! currently we do support PHP, Node.js (if you turn on "Use custom web server" in an HTML pad), and Python Flask ([shown here](https://codepad.app/edit/796qh3gz)), but it's still tough to set up so we should have dedicated project types for servers.
